modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -2859,38 +2919,23 @@ public void testUpdateObjectAction() throws Exception {
 		_objectActionLocalService.deleteObjectAction(systemObjectAction);
 	}
 
-	private void _addModelResourcePermissions(
-			String objectActionName, long objectEntryId, long userId)
-		throws Exception {
-
-		_resourcePermissionLocalService.addModelResourcePermissions(
-			TestPropsValues.getCompanyId(), TestPropsValues.getGroupId(),
-			userId, _objectDefinition.getClassName(),
-			String.valueOf(objectEntryId),
-			ModelPermissionsFactory.create(
-				HashMapBuilder.put(
-					RoleConstants.USER, new String[] {objectActionName}
-				).build(),
-				_objectDefinition.getClassName()));
-	}
-
-	private ObjectAction _addNotificationTemplateObjectAction(
-			String objectActionTriggerKey, ObjectDefinition objectDefinition)
+	private NotificationTemplate _addEmailNotificationTemplate(
+			String body, String description, String name,
+			long objectDefinitionId, String subject, long userId)
```

Comment:
@algobob reorder to `long userId, long objectDefinitionId, String body, String description, String name, String subject` , following the same order from [here](https://github.com/liferay-bpm/liferay-portal/blob/186bff6a2c9c060cc415230d4989bd85a84ab514/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java#L2931-L2938).